### PR TITLE
Adds annual goals and working group notes

### PR DESCRIPTION
**Pull request by Frijol**
r? @johnnyman727 @rwaldron @HipsterBrown @tcr

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -18,15 +18,50 @@ This repository is for the organization and governance of the Tessel project as
 
 For an introduction to the project's organization and key repos, check out the [Technical Overview](https://tessel.gitbooks.io/t2-docs/content/Debugging/Technical_Overview.html).
 
-## How can I get involved?
+# Current goals
 
-### Quick start
+On September 2, 2016, the Steering Committee set year-long goals. They are organized into two overarching goals, each with several supporting goals:
```
Comment (tcr) [inline README.md]: Probably say "The Tessel Proejct has these goals for September 2017 (our next Tessel Summit)."

Also remove the overuse of "goals" in this paragraph.

File: README.md [hunk 2]
```diff
@@ -18,15 +18,50 @@ This repository is for the organization and governance of the Tessel project as
...
+2. **Demonstrate projects and create new tools that prove Tessel is a production-ready (robust, reliable, and extensible) platform for modern hardware development**
+  * [ ] Create a guide to production-scale deployment of a Tessel project
+  * [ ] Research user needs: What do people need to use T2 in production? Why aren't they using it currently?
+  * [ ] Build Reach (https://github.com/tessel/project/issues/142)
+  * [ ] Get to JavaScript parity with Rust API and documentation (https://github.com/tessel/tessel-rust). Figure out JS-Rust inter-exection
+  * [ ] Investigate and possibly execute a Tessel hardware upgrade (Tessel 2.1) for more RAM and Flash. This should legitimize the platform and build out the capabilities so there is less user experience friction
+
+## Working groups
+
+To enable the above goals, the Tessel Project is establishing working groups.
```
Comment (tcr) [inline README.md]: Change "is establishing" to something that will be current. "The Tessel Project forms working groups to accomplish our year-long goals."

File: README.md [hunk 3]
```diff
@@ -18,15 +18,50 @@ This repository is for the organization and governance of the Tessel project as
...
+
+### What working groups do we need to accomplish this year's goals?
+* **Website WG**:  Create a better website for what we are & what we plan to be based on this year’s goals
+* **Rust WG**: Get Rust to 1st class support
+* **Reach WG**: ship Reach
+* **Learning WG**: investigate user needs for production & write a guide on how to take a Tessel project to product scale. Also build and document GPIO/GUTS (Great Uses for Tessel in Stuff e.g. hacking a production system) projects
+* **Tessel space WG**: make larger projects deployable on T2, possibly as a 2.1 Tessel hardware
+
+### How do we implement these WGs?
+We don't want to kick off all 5 at once, so we've picked 2 as pilots: website WG and Learning WG. These two are selected because they are reasonably well defined, accessible, and don't require specialized skills.
```
Comment (tcr) [inline README.md]: "We have two pilot working groups: the Website WG and the Learning WG."

File: README.md [hunk 4]
```diff
@@ -18,15 +18,50 @@ This repository is for the organization and governance of the Tessel project as
...
+
+We will bring up the other WGs as we become comfortable with the process.
```
Comment (tcr) [inline README.md]: (Other WGs will be formed as we learn from working group progress and as interested contributors come on board. Reach out to us if you're interested!)

File: README.md [hunk 5]
```diff
@@ -18,15 +18,50 @@ This repository is for the organization and governance of the Tessel project as
...
+
+# How can I get involved?
```
Comment (tcr) [inline README.md]: "# How can I get involved with the Tessel Project?"

Better phrasing of what this question answers

**Comment by HipsterBrown** [comment]
LGTM :+1:

**Pull request merged**